guides/Guide to Opening Your First PR.md

```diff
@@ -0,0 +1,91 @@
+Guide to Opening Your First PR
```

Comment:
We have existing guides on the github wiki that go very in-depth. They use the tortoisegit gUI, which ironically I haven't installed on my desktop despite that now being my primary coding machine, but they already exist.

Reply:
Fair, but this one is intended for "help I've literally never touched anything before how do I just pr my one smol change"

Reply:
Hence the very specific, slow step-by-step instructions.